src/Python/ImageData/ImageNormalize.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+import vtk
+
+def main():
+    colors = vtk.vtkNamedColors()
+    # Create an image
```

Comment:
For consistency, please use the same colours as in the Cxx example.
You can define user colours using vtNamedColors e.g call them `left_bkg`, `right_bkg`
``` python
colors.SetColor("left_bkg", [0.6, 0.5, 0.4, 1.0])
colors.SetColor("right_bkg", [0.4, 0.5, 0.6, 1.0])
```

Reply:
@ajpmaclean Sure! I will change as suggested.